features/facebookImport/src/locales/de/miniStories/advertisingValue.json

```diff
@@ -1,5 +1,5 @@
 {
     "title": "Ihr Werbe-Wert",
-    "intro": "Da Facebook sein Geld mit dem Verkauf von Werbung verdient, hat es Ihnen <strong>{{number_interests}}</strong> sogenannte Werbeinteressen zugewiesen, wie",
+    "intro": "Da Facebook sein Geld mit dem Verkauf von Werbung verdient, hat es Dir <strong>{{number_interests}}</strong> sogenannte Werbeinteressen zugewiesen, wie z.B.",
```

Comment:
Dang it, resolved this conflict wrong: It should _not_ change from `Ihnen` zu `Dir`! Someone would need to fix that :P